src/spikeinterface/core/segmentutils.py

```diff
@@ -148,17 +148,17 @@ def __init__(self, recording_list, ignore_times=True, sampling_frequency_max_dif
 class ProxyConcatenateRecordingSegment(BaseRecordingSegment):
     def __init__(self, parent_segments, sampling_frequency, ignore_times=True):
         if ignore_times:
-            d = {}
-            d["t_start"] = None
-            d["time_vector"] = None
-            d["sampling_frequency"] = sampling_frequency
+            time_kwargs = {}
```

Comment:
This also got in my way when debuggin, sorry @samuelgarcia . If you want to keep single letter, feel free to rever to something other than d.